### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/capa/actions/runs/3835816204#summary-10429039666

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/capa/actions/runs/3835816334#summary-10429040004

### Checklist

- [x] No CHANGELOG update needed

- [x] No new tests needed

- [x] No documentation update needed

Note: I understand that you have commit style preferences. If the some of the changes suggested here are acceptable, I'd be happy to rewrite the commit(s) to fit them. Some repositories prefer one change per commit and some prefer one change for the whole series. (Some ask to split things into smaller PRs -- I don't think that will be necessary here, but I could be wrong.)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ This project and everyone participating in it is governed by the [Capa Code of C
 
 ### Capa and its repositories
 
-We host the capa project as three Github repositories:
+We host the capa project as three GitHub repositories:
```
Comment (jsoref) [inline .github/CONTRIBUTING.md:34]: Brand

File: CHANGELOG.md [hunk 2]
```diff
@@ -1300,7 +1300,7 @@ Download a standalone binary below and checkout the readme [here on GitHub](http
   - setup: pin vivisect version @williballenthin
   - setup: bump vivisect dependency version @williballenthin
   - setup: set Python project name to `flare-capa` @williballenthin
-  - ci: run tests and linter via Github Actions @Ana06
+  - ci: run tests and linter via GitHub Actions @Ana06
```
Comment (jsoref) [inline CHANGELOG.md:1303]: Some projects don't like changing changelogs. I'm happy to drop this (or anything else)

File: capa/features/extractors/dnfile/helpers.py [hunk 3]
```diff
@@ -186,7 +186,7 @@ def get_dotnet_managed_methods(pe: dnfile.dnPE) -> Iterator[DnType]:
         Each row represents a class in the current assembly.
             TypeName (index into String heap)
             TypeNamespace (index into String heap)
-            MethodList (index into MethodDef table; it marks the first of a continguous run of Methods owned by this Type)
+            MethodList (index into MethodDef table; it marks the first of a contiguous run of Methods owned by this Type)
```
Comment (jsoref) [inline capa/features/extractors/dnfile/helpers.py:189]: This is halfway between contiguous and continuous. Google Sheets guessed this and I have no idea/opinion, so I left it this way.

File: scripts/capa_as_library.py [hunk 4]
```diff
@@ -16,7 +16,7 @@
 from capa.engine import *
 
 
-# == Render ddictionary helpers
+# == Render dictionary helpers
```
Comment (jsoref) [inline scripts/capa_as_library.py:19]: It's possible this is a thing. I think at one point I convinced myself it was. But I've given up...

File: scripts/setup-linter-dependencies.py [hunk 5]
```diff
@@ -70,7 +70,7 @@ def __init__(self):
         self._memory_store = MemoryStore(stix_data=stix_json["objects"])
 
     @staticmethod
-    def _remove_deprecated_objetcs(stix_objects) -> List[AttackPattern]:
+    def _remove_deprecated_objects(stix_objects) -> List[AttackPattern]:
```
Comment (jsoref) [inline scripts/setup-linter-dependencies.py:73]: This appears to be a private method, so renaming it should be harmless.

(FWIW, the history of `deprecated` is itself amusing, but not relevant here.)

File: tests/test_ida_features.py [hunk 6]
```diff
@@ -37,7 +37,7 @@ def check_input_file(wanted):
 def get_ida_extractor(_path):
     check_input_file("5f66b82558ca92e54e77f216ef4c066c")
 
-    # have to import import this inline so pytest doesn't bail outside of IDA
+    # have to import this inline so pytest doesn't bail outside of IDA
```
Comment (jsoref) [inline tests/test_ida_features.py:40]: This feels like an errant doubling. But it's possible it isn't. (This heuristic is something I manually consider/apply)

**Review by jsoref — COMMENTED** [review]
Most changes suggested by Google Sheets. All fault mine.

**Review by williballenthin — APPROVED** [review]
wow, lots of great fixes here, thank you @jsoref!

File: capa/features/extractors/dnfile/helpers.py (hunk 3 above)
Comment (williballenthin) [reply capa/features/extractors/dnfile/helpers.py:189]: lol, this one is funny

**Comment by mr-tz** [comment]
Neat, thank you very much!

**Pull request merged**